containers/cobald-tardis/hooks/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker build --build-arg SOURCE_BRANCH=$SOURCE_BRANCH --build-arg SOURCE_REPO=$(git remote get-url origin) -f $DOCKERFILE_PATH -t $IMAGE_NAME .
```

Comment:
Where are the environment variables `SOURCE_BRANCH`, `DOCKERFILE_PATH` and `IMAGE_NAME` taken from? Should be somewhere in the CI, right?

Reply:
`SOURCE_BRANCH`, `DOCKERFILE_PATH` and `IMAGE_NAME` environment variables are set by the dockerhub build build process. See https://docs.docker.com/docker-hub/builds/advanced/#environment-variables-for-building-and-testing